company.el

```diff
@@ -237,6 +242,8 @@ The visualized data is stored in `company-prefix', `company-candidates',
                          (const :tag "preview" company-preview-frontend)
                          (const :tag "preview, unique only"
                                 company-preview-if-just-one-frontend)
+                         (const :tag "preview, unique and common part"
```

Comment:
You can just say "common", I think. When there's just one, it'll be the whole candidate.